src/ucp/core/ucp_worker.c

```diff
@@ -144,12 +215,39 @@ static ucs_status_t ucp_worker_add_iface(ucp_worker_h worker,
         }
     }
 
+    /* Set wake-up handlers */
+    if (attr->cap.flags & UCT_IFACE_FLAG_WAKEUP) {
+        status = uct_wakeup_open(iface,
+                                 UCT_WAKEUP_RX_AM | UCT_WAKEUP_RX_SIGNALED_AM,
```

Comment:
i think we need for tx as well - because if something goes to pending we need to make sure it would make progress.